config/default.json

```diff
@@ -95,8 +95,8 @@
     "auth_token": false
   },
   "upload_image": false,
-  "ws_connection_client": "wss://steemd.steemit.com",
-  "ws_connection_server": "wss://steemd.steemit.com",
+  "steemd_connection_client": "wss://steemd.steemit.com",
+  "steemd_connection_server": "wss://steemd.steemit.com",
```

Comment:
Eventually, yes. I think the default should still be what it is currently for now... we'll still be using websockets to steemd and not jussi/api.steemit until we do a thorough public test on staging. Merging this in though will allow us to function either with websockets or http/jsonrpc, and will ensure that other branches being tested on dev/staging will already be using jsonrpc as well. Then it will just be an env var change in production when we actually switch it over (and we can then update the defaults to api.steemit.com in a new branch).